docs/iris/example_code/Meteorology/COP_1d_plot.py

```diff
@@ -81,8 +81,7 @@ def main():
                 label='pre-industrial', lw=1.5)
 
     # Establish where r and t have the same data, i.e. the observations
-    common = np.where(a1b_mean.data == e1_mean.data)[0]
-    observed = a1b_mean[common]
+    observed = a1b_mean[:np.argmin(np.isclose(a1b_mean.data, e1_mean.data))]
```

Comment:
> Out of curiosity, why?

this was some feedback from a colleague who had used our example a number of times in a code review exercise and they felt this was better syntax; which I agree with